Refactoring of server deployment configs

TanVD opened the pull request:
1.Deployment scenario with all tomcat webapps on one tomcat created.
2.Ports and paths of all services now assigned in parent pom.xml

TanVD committed:
Optimizing tomcat
1.Configuration for multiple webapps on one tomcat added
2.Properties added for paths and ports of servers

TanVD committed:
1.CallTomcat.sh updated
2.Reminder to update manually ports and paths in callTomcat.sh added

TanVD committed:
1.Fixes for properties
2.Deployment for one tomcat clean fixed

TanVD committed:
1.CallTomcat.sh fixed according to properties.
2.Codacy fixes

TanVD committed:
Minor style fixes.

TanVD committed:
Property names brought in correspondence with styleguide.

jzuken commented on auth-service/pom.xml:
```diff
@@ -19,6 +19,7 @@
         <resources>
             <resource>
                 <directory>src/main/resources</directory>
+                <filtering>true</filtering>
```
why filtering here?

TanVD committed:
Merge branch 'master' into tomcat-oneforall

TanVD committed:
Maven profiles define tomcat services configs.

From now on properties defining ports, paths and other elements of
tomcat services configuration defined as a profile specific properties
in parent pom.xml. It means that you need explicitly state right
profile while executing maven goal. Use command of such type:

mvn <goal> -P <name-of-profile>

TanVD committed:
Filter one property TS class instead of whole editor app.

Instead of filtering whole editor-service TS app as a resource, now filter
only resources dir of TS app with TS classes used as property files.

jzuken commented on pom.xml:
```diff
@@ -43,6 +28,53 @@
+            </properties>
+        </profile>
+        <profile>
+            <id>OneTomcatForAll</id>
```
let's add 

```
<activation>
      <activeByDefault>true</activeByDefault>
</activation>
```

here to make it default profile

jzuken commented on pom.xml:
```diff
@@ -43,6 +28,53 @@
+    <!-- Profiles state ports and paths only for tomcat based services for now -->
+    <profiles>
+        <profile>
+            <id>Travis</id>
```
is there a reason profile names start with a capital letter?

TanVD replied:
No reason. Will change it to lower-case style.

TanVD committed:
Fix names for profiles and adds default profile.

Now all profiles in maven written according to styleguide for names.
Profile one-tomcat-for-all now is default and will be used if no
profiles were stated in maven command.
